### DISCUSSION TITLE
Red & Green Boxes?

### DISCUSSION THREAD

**Discussion by cwolf908** [kernc/backtesting.py#360] [Q&A] [answered]
I may be losing my mind and skimming over the source of these artifacts on the plot, but I've searched the issue tracker and forum extensively and cannot - for the life of me - figure out why and how the green and red boxes are overlaying on the candlestick chart. They don't seem to correspond to anything in my code. Can anyone please explain?

**Comment by kernc** [comment, marked as answer]
See [`Backtest.plot(superimpose=)`](https://kernc.github.io/backtesting.py/doc/backtesting/backtesting.html#backtesting.backtesting.Backtest.plot).

**Reply by cwolf908** [reply]
Thank you for getting back to me and for the quick reply! I'm still a bit confused, however, as the "bars" being superimposed are of different lengths. I'm using daily data and they are not all monthly bars. Some are wider and done narrower. I'll set it to false, though, and see if that removes them.

**Reply by cwolf908** [reply]
Setting superimpose to false did indeed remove the boxes. Thank you again!
